Upstream-Status: Pending
---
diff --git a/setup.py b/setup.py
index 056dcd97460..c7c23db2ed8 100644
--- a/setup.py
+++ b/setup.py
@@ -882,10 +882,6 @@ setup(
     extras_require=extras_require,
     python_requires=">=3.11",
     keywords="onnx machine learning",
-    entry_points={
-        "console_scripts": [
-            "onnxruntime_test = onnxruntime.tools.onnxruntime_test:main",
-        ]
-    },
+    entry_points={},
     classifiers=classifiers,
 )
