From fdc83b5bc1c0b50641fa022455951f55cefa58b3 Mon Sep 17 00:00:00 2001
From: Nobuo Tsukamoto <nextremer.nb.o@gmail.com>
Date: Tue, 30 Apr 2024 20:09:54 +0900
Subject: [PATCH] Fixed onnxruntime QA Issue. (#1)

Upstream-Status: Pending
---
 setup.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/setup.py b/setup.py
index 58aadc75b5..f110b0d5ef 100644
--- a/setup.py
+++ b/setup.py
@@ -912,10 +912,6 @@ setup(
     extras_require=extras_require,
     python_requires=">=3.11",
     keywords="onnx machine learning",
-    entry_points={
-        "console_scripts": [
-            "onnxruntime_test = onnxruntime.tools.onnxruntime_test:main",
-        ]
-    },
+    entry_points={},
     classifiers=classifiers,
 )
