Upstream-Status: Pending
---
 cmake/external/onnxruntime_external_deps.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/external/onnxruntime_external_deps.cmake b/cmake/external/onnxruntime_external_deps.cmake
index f523fefdd1..da22c79a7e 100644
--- a/cmake/external/onnxruntime_external_deps.cmake
+++ b/cmake/external/onnxruntime_external_deps.cmake
@@ -610,6 +610,7 @@ if(onnxruntime_ENABLE_DLPACK)
     EXCLUDE_FROM_ALL
     FIND_PACKAGE_ARGS NAMES dlpack
   )
+  set(CMAKE_POLICY_VERSION_MINIMUM 3.5)
   onnxruntime_fetchcontent_makeavailable(dlpack)
 endif()
 
