Upstream-Status: Pending
---
 cmake/onnxruntime_python.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmake/onnxruntime_python.cmake b/cmake/onnxruntime_python.cmake
index c89f8845a1..88382ee0e0 100644
--- a/cmake/onnxruntime_python.cmake
+++ b/cmake/onnxruntime_python.cmake
@@ -235,7 +235,6 @@ target_link_libraries(onnxruntime_pybind11_state PRIVATE
     onnxruntime_common
     onnxruntime_flatbuffers
     ${pybind11_lib}
-    Python::NumPy
 )
 
 # The CUDA quantization helpers (pack_weights_for_cuda_mixed_gemm) are built into a
