Upstream-Status: Pending
---
 cmake/ortlib.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/ortlib.cmake b/cmake/ortlib.cmake
index 92da523f..03282ea4 100644
--- a/cmake/ortlib.cmake
+++ b/cmake/ortlib.cmake
@@ -97,7 +97,7 @@ if(ORT_HOME)
     set(ORT_HEADER_DIR ${ORT_HOME}/include/onnxruntime)
     set(ORT_LIB_DIR ${ORT_HOME}/lib)
   else()
-    set(ORT_HEADER_DIR ${ORT_HOME}/include)
+    set(ORT_HEADER_DIR ${ORT_HOME}/include/onnxruntime)
     set(ORT_LIB_DIR ${ORT_HOME}/lib)
   endif()
 else()
