Upstream-Status: Pending
---
 cmake/target_platform.cmake | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/cmake/target_platform.cmake b/cmake/target_platform.cmake
index 222e4205..c7dd111e 100644
--- a/cmake/target_platform.cmake
+++ b/cmake/target_platform.cmake
@@ -59,6 +59,10 @@ else()
     set(genai_target_platform "x64")
   elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "powerpc")
     set(genai_target_platform "powerpc")
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "riscv64")
+    set(genai_target_platform "riscv64")
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "riscv32")
+    set(genai_target_platform "riscv32")
   else()
     message(FATAL_ERROR "Unsupported architecture. CMAKE_SYSTEM_PROCESSOR: ${CMAKE_SYSTEM_PROCESSOR}")
   endif()
